### PR TITLE
[pcl_ros] Use timer to publish cloud periodically

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by wkentaro** [comment]
ping

**Comment by wkentaro** [comment]
ping

File: pcl_ros/tools/pcd_to_pointcloud.cpp [hunk 1]
```diff
@@ -89,46 +92,29 @@ class PCDGenerator
       if (file_name_ == "" || pcl::io::loadPCDFile (file_name_, cloud_) == -1)
         return (-1);
       cloud_.header.frame_id = tf_frame_;
+      // set timer to publish cloud periodically
+      timer_ = nh_.createWallTimer(
```
Comment (k-okada) [inline pcl_ros/tools/pcd_to_pointcloud.cpp]: does this synchronize with ros clock?

Comment (wkentaro) [inline pcl_ros/tools/pcd_to_pointcloud.cpp]: thanks for the comment, it should be `nh_.createTimer`

Comment (wkentaro) [inline pcl_ros/tools/pcd_to_pointcloud.cpp]: http://wiki.ros.org/roscpp/Overview/Timers

**Comment by wkentaro** [comment]
I have updated the commit to use `createTimer` instead of `createWallTimer`

**Comment by wkentaro** [comment]
ping

**Comment by paulbovbel** [comment]
If this is covered by your other PR as well, let's close this one?

**Comment by wkentaro** [comment]
Ok!

**Pull request closed without merging**